config.js

```diff
@@ -398,6 +398,11 @@ module.exports.TRAINING_KUBERNETES_POD_SPEC_OVERRIDE = {};
 */
 module.exports.TRAINING_KUBERNETES_CONTAINER_SPEC_OVERRIDE = {};
 
+/**
+  S3 directory where tensboard events are sycned to during training.
+*/
+module.exports.TENSORBOARD_DIR = null;
```

Comment:
This can be a file:/// URL too right? In case one wants to run everything on one machine?
Also, typo "sycned"

Reply:
Done.